_posts/2021-12-31-a-year-in-review-2021.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: epic
+title: "A Year in Review: 2021"
+date: 2022-01-06
+categories: [team, engineering]
+author: artsy_engineering
+---
+
+Ufff, 2021 was a tough year. It was the second year of the covid-19 pandemic. We tried our best to support each
+other through these challenging times and in that spirit, we feel like it's important not to lose sight of our
+accomplishments. As we say goodbye to 2021 and hello to 2022, here are some Artsy Engineering wins from this past year.
+What did you ship in 2021? We would love to hear about it and celebrate with you!
+
+<!-- more -->
+
+- Recorded and released 42 episodes of the [Artsy Engineering Radio][]
+- Onboarded 23 engineers in Berlin, Germany
+- Leveraged our investment in React Native to launch the [Artsy Android App][]
```

Comment:
O. M. G. I had no idea… The plan, it has finally come together.

![](https://media.giphy.com/media/1naY1TdNLTlSEdzi5D/giphy.gif)

Y'all are the best ❤️